superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -454,6 +462,13 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
   const [dashboardOptions, setDashboardOptions] = useState<MetaObject[]>([]);
   const [chartOptions, setChartOptions] = useState<MetaObject[]>([]);
   const [tabOptions, setTabOptions] = useState<TabNode[]>([]);
+  const [nativeFilterOptions, setNativeFilterOptions] = useState<object>([]);
+  const [nativeFilterValues, setNativeFilterValues] = useState<object>([]);
+  const [tabNativeFilters, setTabNativeFilters] = useState<object>({});
```

Comment:
I think the name might be confusing here. It seems this is hosting all native filters independently of the current tab?